tools/static-assets/skel-svelte/imports/ui/counter.svelte.js

```
@@ -0,0 +1,5 @@
+let counter = $state(0);
+
+export const getCounter = () => counter;
+
+export const addToCounter = () => { counter += 1 };
```

ESLint

[error] Replace `·counter·+=·1·` with `⏎··counter·+=·1;⏎`

(prettier/prettier)